examples/sequence.rb

```diff
@@ -0,0 +1,37 @@
+require 'dotenv'
+Dotenv.load '.env-fixer_client'
+
+require 'fixer_client'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.